.circleci/config.yml

```diff
@@ -18,8 +18,6 @@ jobs:
       - run:
           name: Deploy to cloud.gov
           filters:
```

Comment:
[nitpick]: This should probably be removed too since there's nothing to filter.

Reply:
oh whoops! meant to delete that. will do :)